src/Microsoft.ML.Data/Scorers/PredictionTransformer.cs

```diff
@@ -36,17 +36,18 @@ public abstract class PredictionTransformerBase<TModel> : IPredictionTransformer
 
         public TModel Model { get; }
 
-        public PredictionTransformerBase(IHost host, TModel model, ISchema trainSchema, string featureColumn)
+        public PredictionTransformerBase(IHost host, TModel model, ISchema trainSchema, string featureColumn = null)
```

Comment:
> = null [](start = 108, length = 7)

nope, that should still be required param. Just 'null' should now be acceptable.